dist/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Restaurant</title>
+</head>
+<body>
+    <nav class="tabs">
+        <button class="navbar-btn">Home</button>
+        <button class="navbar-btn">Menu</button>
+        <button class="navbar-btn">Contact</button>
+    </nav>
```

Comment:
- [ ] In here, you have some HTML content containing the navbar links. Remember the requirement says all HTML content (except `html`, `body` and `<div id="content">` ) should be dynamically generated with javascript and which includes the nav content too. Kindly ensure you adjust your code to the above requirement. You can consider creating a module for the navbar content and calling it in the index.js on page load. Here is a [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/append_requirement.png) for clarification.